patch_release_main.nf

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env nextflow
+// Ensure DSL2
+nextflow.enable.dsl = 2
+
+// IMPORT MODULES
+include { patch_release } from './modules/patch_release'
+include { create_data_guide } from './modules/create_data_guide'
+include { create_dashboard_html } from './modules/create_dashboard_html'
+
+params.release_synid = "syn53170398"  // 15.4-consortium
```

Comment:
Do we want to make these configurable on Tower? Along with description of what they should be. I think there's a way to modify `nextflow_schema.json` and `nextflow.config` to handle when we might have multiple sets of parameters.

Like create a param called pipeline_type or release_type, and in our nextflow.config have different statements that 

Example)
```
params.release_type = "patch"

if (params.release_type == 'patch') {
    ...
} else if (params.pipeline == 'pipelineB') {
   ...
}

process {
    withName: 'processA' {
        when: params.release_type == "patch"
        ...
    }
```

Just hypothetical, I haven't played around with nextflow enough to see

Reply:
Thats a great point, it would be ideal to do that!

Reply:
There isn't a way to do that, but I did add a patch release schema!